generators/server/templates/pom.xml.ejs

```diff
@@ -1061,7 +1061,7 @@
                                 <SPRING_OUTPUT_ANSI_ENABLED>ALWAYS</SPRING_OUTPUT_ANSI_ENABLED>
                                 <JHIPSTER_SLEEP>0</JHIPSTER_SLEEP>
                             </environment>
-                            <useCurrentTimestamp>true</useCurrentTimestamp>
+                            <creationTime>USE_CURRENT_TIMESTAMP</useCurrentTimestamp>
```

Comment:
you missed closing tag. This is failing tests:
 https://dev.azure.com/jhipster/generator-jhipster/_build/results?buildId=5764&view=logs&jobId=156bdf7b-70dc-51a2-f964-752e46e62347&taskId=b7e9856b-ece4-51e3-a338-586b2e499c39&lineStart=16&lineEnd=17&colStart=124&colEnd=4

Reply:
Thx